### ISSUE TITLE
Bare ipv6 addresses don't work

### ISSUE THREAD

**Issue by akhepcat** [arthepsy/ssh-audit#26]
(address changed to protect the innocent)

    ./ssh-audit.py -6 2000:100:100:1::3
    [exception] [Errno -9] Address family for hostname not supported
    
    ./ssh-audit.py -6 [2000:100:100:1::3]
    [exception] [Errno -2] Name or service not known
    
    ./ssh-audit.py -6 "[2000:100:100:1::3]"
    [exception] [Errno -2] Name or service not known

However, using a name (whether dns or /etc/hosts)  works correctly.

**Comment by arthepsy** [comment]
Yes, this is bug. Thanks for the report, will fix this. For now, as a workaround, You could use:
```./ssh-audit.py -p 22 2000:100:100:1::3```

**Comment by jtesta** [comment]
@akhepcat This should be fixed in my v2.0.0 release: https://github.com/jtesta/ssh-audit/releases/tag/v2.0.0
